shell/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -84,6 +91,10 @@ public final void before() throws Exception {
     mNewOutput = new PrintStream(mOutput);
     mOldOutput = System.out;
     System.setOut(mNewOutput);
+    // clear the loginUser
+    Field field = LoginUser.class.getDeclaredField("sLoginUser");
```

Comment:
To set a `null` value, you'll need to cast it to the right type. Take a look at http://stackoverflow.com/questions/5385161/powermock-testing-set-static-field-of-class